### PR TITLE
[Merged by Bors] - feat: `Disjoint G₁.edgeFinset G₂.edgeFinset ↔ Disjoint G₁ G₂`

### PR DESCRIPTION
**Pull request by YaelDillies**
From LeanCamCombi

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Finite.lean [hunk 1]
```diff
@@ -92,6 +92,15 @@ theorem edgeFinset_inf [DecidableEq V] : (G₁ ⊓ G₂).edgeFinset = G₁.edgeF
 theorem edgeFinset_sdiff [DecidableEq V] :
     (G₁ \ G₂).edgeFinset = G₁.edgeFinset \ G₂.edgeFinset := by simp [edgeFinset]
 
+lemma disjoint_edgeFinset : Disjoint G₁.edgeFinset G₂.edgeFinset ↔ Disjoint G₁ G₂ := by
```
Comment (b-mehta) [inline Mathlib/Combinatorics/SimpleGraph/Finite.lean:95]: simp on any or all of these?

**Review by b-mehta — APPROVED** [review]
bors d+

File: Mathlib/Combinatorics/SimpleGraph/Finite.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Combinatorics/SimpleGraph/Finite.lean:95]: No, because `edgeFinset` is an abbrev (!)

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**